cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -837,5 +839,308 @@ define([
             view.collection.add({name: 'Editing Group', editing: true});
             verifyEditingGroup(view, true);
         });
+
+    });
+
+    describe('Content groups details view', function() {
+
+        beforeEach(function() {
+            TemplateHelpers.installTemplate('content-group-details', true);
+            this.model = new GroupModel({name: 'Content Group', id: 0});
+
+            var saveableModel = new GroupConfigurationModel({
+                name: 'Content Group Configuration',
+                id: 0,
+                scheme:'cohort',
+                groups: new GroupCollection([this.model]),
+            }, {canBeEmpty: true});
+
+            saveableModel.urlRoot = '/mock_url';
+
+            this.collection = new GroupConfigurationCollection([ saveableModel ]);
+            this.collection.outlineUrl = '/outline';
+
+            this.view = new ContentGroupDetailsView({
+                model: this.model
+            });
+            appendSetFixtures(this.view.render().el);
+        });
+
+        it('should render properly', function() {
+            expect(this.view.$el).toContainText('Content Group');
+            expect(this.view.$el).toContainText('ID: 0');
+            expect(this.view.$('.delete')).toExist();
+        });
+
+        it('should show empty usage appropriately', function() {
```

Comment:
It looks like most of these tests are copies of the original tests for content experiment groups. Can you refactor the code so that the logic is shared rather than duplicated. Probably the simplest way is to have helper methods for common code, but you might be able to have a single test that handles both cases one after the other. I think the former is probably clearer as it is explicit about what it is testing.